src/components/Icons/ChevronDown.js

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+const SvgChevronDown = ({ title, titleId, ...props }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="2em" height="2em" fill="none" viewBox="0 0 24 24" aria-labelledby={titleId} {...props}>
+    {title ? <title id={titleId}>{title}</title> : null}
```

Comment:
You can also write it like this:
{title && <title id={titleId}>{title}</title> }

Anton pointed this out to me yesterday🙃